src/applications/vaos/covid-19-vaccine/flow.js

```diff
@@ -8,7 +8,7 @@ import {
 
 export default {
   home: {
-    url: '/',
+    url: '/new-appointment',
```

Comment:
This makes it so the back button on the plan ahead page goes to the start of the new appointment flow